--- library.js.orig     2020-05-18 17:14:18.682328912 +0200
+++ library.js  2020-05-18 17:14:48.366639562 +0200
@@ -271,7 +271,7 @@
     // pid_t fork(void);
     // http://pubs.opengroup.org/onlinepubs/000095399/functions/fork.html
     // We don't support multiple processes.
-    setErrNo({{{ cDefine('EAGAIN') }}});
+    setErrNo({{{ cDefine('ENOTSUP') }}});
     return -1;
   },
   vfork: 'fork',
@@ -696,7 +696,7 @@
     // http://pubs.opengroup.org/onlinepubs/000095399/functions/system.html
     // Can't call external programs.
     if (!command) return 0; // no shell available
-    setErrNo({{{ cDefine('EAGAIN') }}});
+    setErrNo({{{ cDefine('ENOTSUP') }}});
     return -1;
   },
